Added my resume - Emzzy241.md

Emzzy241 opened the pull request:
I added my resume in Markdown format as per the contest instructions.

In my resume, I have detailed my past experiences and projects I have worked on/built.

Thank you for this amazing opportunity.

Emzzy241 committed:
Added my resume - Emzzy241.md

Emzzy241 commented:
Check out my recent PR, @Bhopal-DAO

Eshan-Sharma commented on Emzzy241.md:
```diff
@@ -0,0 +1,73 @@
+# Emmanuel Mojiboye
+
+📍 Ifako-Ijaiye, Lagos-Nigeria  
+📞 +2349012857304  
```
I would suggest do not include phone number in public repo unless you are okay with some spam

Eshan-Sharma commented on Emzzy241.md:
```diff
@@ -0,0 +1,73 @@
+
+---
+
+## 🚀 Projects & Achievements
```
Add live links if available

Eshan-Sharma commented on Emzzy241.md:
```diff
@@ -0,0 +1,73 @@
+
+---
+
+## 💼 Work Experience
```
Be a little more objective in your description of experience, add x% increase in efficiency, y% decrease in latency while working on backend etc.

Emzzy241 replied:
Thank you very very much, I would apply the changes you suggested.

Regarding the phone number, I think I am cool with it

Emzzy241 committed:
Update README based on Collaborator's feedbacks

Eshan-Sharma reviewed (approved):
Good work, approved

Emzzy241 commented:
I have updated my resume based on the feedback provided.

- Added measurable impact to work experience.
- Included live links where available.
- Removed my phone number.

Please review the changes and let me know if anything else needs improvement. Thanks!

[Emzzy241.md](https://github.com/user-attachments/files/18652222/Emzzy241.md)
